lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -146,6 +147,89 @@ class ConvertAten_IndexPutImplOp
   }
 };
 
+class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
+public:
+  using OpConversionPattern<AtenConvolutionOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenConvolutionOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    SmallVector<Type> resultTypes;
+    if (failed(OpConversionPattern<AtenConvolutionOp>::getTypeConverter()
+                   ->convertTypes(op->getResultTypes(), resultTypes))) {
+      return failure();
+    }
+
+    SmallVector<Value> operands;
+    SmallVector<StringRef> operandNames;
+
+    auto addOperand = [&](std::string name, Value value) {
+      operandNames.push_back(name);
+      operands.push_back(value);
+    };
+
+    addOperand("input", adaptor.getInput());
+    addOperand("weight", adaptor.getWeight());
+    if (!adaptor.getBias().getType().isa<Torch::NoneType>()) {
+      addOperand("bias", adaptor.getBias());
+    }
+
+    SmallVector<NamedAttribute> attrs;
+
+    attrs.push_back(rewriter.getNamedAttr(
+        "torch_operand_names", rewriter.getStrArrayAttr(operandNames)));
+
+    auto addListOfIntAttr = [&](const std::string &name, Value value) {
+      SmallVector<int64_t> valueInt;
+      if (!matchPattern(adaptor.getStride(),
+                        m_TorchListOfConstantInts(valueInt)))
```

Comment:
I think this has a bug. Should this be
```
if (!matchPattern(value, m_TorchListOfConstantInts(valueInt)))
```
instead?

Reply:
Yikes :( Fixed.